doc/api/dns.md

```diff
@@ -195,6 +196,31 @@ Uses the DNS protocol to resolve mail exchange records (`MX` records) for the
 contain an array of objects containing both a `priority` and `exchange`
 property (e.g. `[{priority: 10, exchange: 'mx.example.com'}, ...]`).
 
+## dns.resolveNaptr(hostname, callback)
+
+Uses the DNS protocol to resolve regular expression based records (`NAPTR`
+records) for the `hostname`. The `addresses` argument passed to the `callback`
+function will contain an array of objects with the following properties:
+
+* `flags`
+* `service`
+* `regexp`
+* `replacement`
+* `order`
+* `preference`
+
+```
```

Comment:
nit: Can you add a short `For example:` in here and tack a `js` to the end of the three backticks to get the code styling. Thank you!